packages/core/types/tests-theming.tsx

```diff
@@ -139,7 +87,4 @@ const style2: ObjectStyleDefinition = {
 }
 
 // Can use ThemeProvider
-;<ThemeProvider theme={{ prop: 'val' }} />
-;<TypedThemeProvider theme={{ primary: '', secondary: '' }} />
-// $ExpectError
-;<TypedThemeProvider theme={{ nope: 'string' }} />
+;<ThemeProvider theme={{ primary: 'val' }} />
```

Comment:
How this works now? Is `{primary: string}` compatible with builtin (empty) `Theme`?

Reply:
ah, you extend it in the other file, cool 👍